10-inference.Rmd

```diff
@@ -210,41 +210,29 @@ the mean of the sample proportions.
 
 <!--- > **Note:** If random samples of size $n$ are taken from a population, $\hat{p}$ will be approximately Normal with mean $p$ and standard deviation $\sqrt{\frac{p(1-p)}{n}}$ as long as the sample size $n$ is large enough such that $np$ and $n(1 - p)$ are at least 10, where $p$ is the population proportion, $\hat{p}$ is the sample proportion and $n$ is the sample size. 
 --->
 
 ### Sampling distributions for means 
 
-In the previous section, our variable of interest&mdash;Timbit flavour&mdash;was
-*categorical*, and the population parameter of interest was the proportion of chocolate
-Timbits. As mentioned in the introduction to this chapter, there are many choices of population parameter for each type of observed variable. What if we wanted to infer something about a population of *quantitative* variables instead? For instance, a traveller visiting Vancouver, BC may wish to know about the prices of staying somewhere using Airbnb, an online marketplace for arranging places to stay. Particularly, they might be interested in estimating the population mean price per night of Airbnb listings in Vancouver, BC. This section will study the case where we are interested in the population mean of a quantitative variable.
-
-We will look at an example using data from [Inside Airbnb](http://insideAirbnb.com/). The data set contains Airbnb listings for Vancouver, Canada, in September 2020. Let's imagine (for learning purposes) that our data set represents the population of all Airbnb rental listings in Vancouver, and we are interested in the population mean price per night.
-Our data contains an ID number, neighbourhood,
-type of room, the number of people the rental accommodates, number of bathrooms, bedrooms, beds, and the price per night.
-
-```{r 11-example-means1, echo = FALSE, message = FALSE, warning = FALSE}
-airbnb <- read_csv("data/listings.csv") %>%
-  select(id, neighbourhood = neighbourhood_cleansed, room_type, accommodates, bathrooms = bathrooms_text, bedrooms, beds, price) %>%
-  mutate(price = as.numeric(str_remove(price, "[$]"))) %>%
-  na.omit()
-airbnb <- airbnb %>%
-  mutate(id = 1:nrow(airbnb))
-head(airbnb)
-```
+In the previous section, our variable of interest&mdash; `room_type` &mdash;was
+*categorical*, and the population parameter of interest was the proportion of listings where the room type was "entire home/apt". As mentioned in the chapter introduction, there are many choices of the population parameter for each type of observed variable. What if we wanted to infer something about a population of *quantitative* variables instead? For instance, a traveller visiting Vancouver, BC, may wish to know about the prices of staying somewhere using Airbnb. Particularly, they might be interested in estimating the population mean price per night of Airbnb listings in Vancouver, BC. This section will study the case where we are interested in the population mean of a quantitative variable.
```

Comment:
again, motivation is missing. traveller might want to know avg prices why? e.g. maybe they can tell whether a particular listing is overpriced or something like that. or maybe city of van needs to know it for some reason (setting tax or something? not sure).